src/rouge.jl

```diff
@@ -0,0 +1,149 @@
+#= ROUGE score implementation
```

Comment:
I don't really like multiline comments `#=`. Many tools find it hard to work with these. I'd prefer to change these to have a `#` on every line. But don't do it by hand, your editor should have some command to comment a block together. I know Juno has that.